spec/javascript/components/supplemental_application/__snapshots__/SupplementalApplicationPage.test.js.snap

```diff
@@ -5908,9 +5890,7 @@ exports[`SupplementalApplicationPage should render the status update dropdown bu
                                             "flagged_applications": Array [],
                                             "general_lottery": true,
                                             "general_lottery_rank": null,
-                                            "has_ada_priorities_selected": Object {
-                                              "": true,
```

Comment:
This is what the bug was with ADA values on the supp app. this extra empty string key was floating around